cli/README.md

```diff
@@ -44,8 +44,8 @@ RABBITMQ_PORT=7008
 RABBITMQ_USERNAME=rabbitmq_username
 RABBITMQ_PASSWORD=rabbitmq_password
 
-POSTGRES_HOST=localhost
-POSTGRES_PORT=7001
+POSTGRES_PIXL_DB_HOST=localhost
+POSTGRES_PIXL_DB_PORT=7001
```

Comment:
How about `EXTERNAL_PIXL_DB_HOST` or `CLI_PIXL_DB_HOST` etc?

Reply:
Sticking with `POSTGRES_ORTHANC_RAW_DB_PORT` and `POSTGRES_PIXL_DB_HOST`. Will explain in comment.